### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by sunnymoon155** [Escanor1996/Speech-Emotion-Recognition-SER-#1]
Hi, thanks for the sharing!
Could you please provide SER_densenet121_partial.pt and SER_densenet121_full.pt in the code?

**Comment by mshivasharan** [comment]
Hello, I am looking for the same. Could you please upload SER_densenet121_full.pt please?

**Comment by levtelyatnikov** [comment]
Hello, I am looking for the same. Could you please upload SER_densenet121_full.pt please?

**Comment by leijue222** [comment]
Can you tell us what the format of the directory is, so that you can retrain your code wiht [dataset1 ](https://www.kaggle.com/dataset/2a4541d696a3fd076152467ace40a7bfe6e85e108f17292df04a4e6d7b4aecaa)and [dataset2](https://www.kaggle.com/dataset/4e97bf2fb36d96647422054bfe9e0bdd34397120fb38eaf8f87c20f243acd511).

**Comment by Escanor1996** [comment]
you can edit your directory in training notebook during creation of the dataloader.

**Comment by leijue222** [comment]
> you can edit your directory in training notebook during creation of the dataloader.

Thanks.
I use the datasets_spectrograms pre-processed appropriately by George Christopoulos.
It worked, but it can't reach 98% accuracy like the jupyter log. 
I only reach 72% accuracy by `SER_TRAINING_Full_dataset.ipynb`.
The accuracy difference is too big. May I have any details missing?

**Comment by NewEricWang** [comment]
@leijue222 , have you  reproduced the reported results (98%)?

**Comment by leijue222** [comment]
> @leijue222 , have you reproduced the reported results (98%)?

I can't...

**Comment by NewEricWang** [comment]
Is the data you used downloaded from the address: http://www.kaggle.com/dataset/4e97bf2fb36d96647422054bfe9e0bdd34397120fb38eaf8f87c20f243acd511 ?

**Comment by NewEricWang** [comment]
@leijue222 ,
Is the data you used downloaded from the address: http://www.kaggle.com/dataset/4e97bf2fb36d96647422054bfe9e0bdd34397120fb38eaf8f87c20f243acd511 ?

**Comment by leijue222** [comment]
Yes

**Comment by NewEricWang** [comment]
@leijue222 , How do you split train set, validation set and test set?  
I find there is not any useful information in the  provided dataset.

**Comment by leijue222** [comment]
Sorry, I already forgot the details about this project. 
According to my investigation, the accuracy of emotion recognition tasks cannot be achieved with a good accuracy rate by using voice or visual information alone. Multimodal emotion perception is a trend.

**Comment by NewEricWang** [comment]
Ok, Thank you!
